core/unix/src/TUnixSystem.cxx

```diff
@@ -744,6 +744,10 @@ Int_t TUnixSystem::GetCryptoRandom(void *buf, Int_t len)
    return len;
 #elif defined(R__GETRANDOM_CLIB)
    return getrandom(buf, len, GRND_NONBLOCK);
+#elif defined(R__USE_URANDOM)
+   std::ifstream urandom{"/dev/urandom"};
+   urandom.read(reinterpret_cast<char *>(buf), len);
```

Comment:
We should check for failure with `if (!urandom) {...}`

Reply:
Thanks! done